style.css

```diff
@@ -150,9 +150,26 @@ main img {
         width: 50%;
         margin: 0 auto;
     }
+
```

Comment:
Why do you keep adding newlines? Please use `git diff` before commiting to make sure you're not adding them.